main.py

```diff
@@ -646,8 +644,12 @@ def update_port_status(self, port_status, source):
         event = KytosEvent(name=event_name, content=content)
         self.controller.buffers.app.put(event)
 
-        msg = 'The port %s from switch %s was %s.'
-        log.debug(msg, port_status.desc.port_no, source.switch.id, status)
+        # pylint: disable=protected-access
+        state_desc = {v: k for k, v in PortState._enum.items()}
+        # pylint: enable=protected-access
+        state = state_desc.get(port.state.value, port.state.value)
```

Comment:
@Ktmi, great suggestion! Indeed, I should've created an issue to enhance this in the future because the way it is being done now is not ideal. Ideally, we should inherit from Enum. I've mapped this improvement on issue kytos-ng/python-openflow#39

The currently solution was tested using Kytos docker image (amlight/kytos:latest) which runs on Python 3.9:

```
# grep PortSta /var/log/syslog
Jan 17 20:25:13 4acfef79ca29 2023-01-17 20:25:13,567 kytos.napps.kytos/of_core:INFO main:630:  PortStatus modified interface 00:00:00:00:00:00:00:03:2 state 0
Jan 17 20:25:13 4acfef79ca29 2023-01-17 20:25:13,576 kytos.napps.kytos/of_core:INFO main:630:  PortStatus modified interface 00:00:00:00:00:00:00:03:2 state OFPPS_LINK_DOWN
Jan 17 20:25:13 4acfef79ca29 2023-01-17 20:25:13,579 kytos.napps.kytos/of_core:INFO main:630:  PortStatus modified interface 00:00:00:00:00:00:00:02:3 state 0
Jan 17 20:25:13 4acfef79ca29 2023-01-17 20:25:13,583 kytos.napps.kytos/of_core:INFO main:630:  PortStatus modified interface 00:00:00:00:00:00:00:02:3 state OFPPS_LINK_DOWN
Jan 17 20:25:17 4acfef79ca29 2023-01-17 20:25:17,613 kytos.napps.kytos/of_core:INFO main:630:  PortStatus modified interface 00:00:00:00:00:00:00:03:2 state OFPPS_LIVE
Jan 17 20:25:17 4acfef79ca29 2023-01-17 20:25:17,614 kytos.napps.kytos/of_core:INFO main:630:  PortStatus modified interface 00:00:00:00:00:00:00:02:3 state OFPPS_LIVE
```